code/datums/components/surgery_initiator.dm

```diff
@@ -100,7 +100,7 @@
 					return
 			else if(carbontarget && surgeryinstance_notonmob.requires_bodypart)
 				return
-			if(surgeryinstance_notonmob.lying_required && (livingtarget.mobility_flags & MOBILITY_STAND))
+			if(surgeryinstance_notonmob.lying_required && livingtarget.body_position != LYING_DOWN)
```

Comment:
```suggestion
			if(surgeryinstance_notonmob.lying_required && livingtarget.body_position == STANDING_UP)
```

Reply:
This is the same as before but less safe if more body positions are added.